src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets

```diff
@@ -51,8 +51,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     <BlazorCacheBootResources Condition="'$(BlazorCacheBootResources)' == ''">true</BlazorCacheBootResources>
 
     <_TargetingNET80OrLater>$([MSBuild]::VersionGreaterThanOrEquals('$(TargetFrameworkVersion)', '8.0'))</_TargetingNET80OrLater>
+    <_TargetingNET100OrLater>$([MSBuild]::VersionGreaterThanOrEquals('$(TargetFrameworkVersion)', '10.0'))</_TargetingNET100OrLater>
     <WasmFingerprintDotnetJs>true</WasmFingerprintDotnetJs>
     <WasmFingerprintDotnetJs Condition="'$(_TargetingNET80OrLater)' == 'true'">false</WasmFingerprintDotnetJs>
+    <WasmFingerprintDotnetJs Condition="'$(_TargetingNET100OrLater)' == 'true' and '$(WriteImportMapToHtml)' == 'true'">true</WasmFingerprintDotnetJs>
```

Comment:
Dumb question, should this be in this SDK?

Reply:
From my point of view, the feature (`WriteImportMapToHtml=true`) doesn't make sense if we don't put fingerprint on dotnet.js. Maybe we can opt-out of setting `WasmFingerprintDotnetJs` here when targeting .NET 10+ and leave it to decide by Wasm SDK, but the logic would the same I think